### DISCUSSION TITLE
Array Depth

### DISCUSSION THREAD

**Discussion by Hetarth02** [laravel/framework#42957] [Ideas]
I have developed an array helper function which returns the maximum depth/nested level of an array.

I have the function method implemented but I do not know what to do next? Is it okay to just generate a PR or are there any steps before it?

Any help would be appreciated. Thanks!

**Comment by henzeb** [comment]
I think you should start there: https://laravel.com/docs/9.x/contributions

**Reply by Hetarth02** [reply]
I have already read it, and following the steps from it I have started a discussion here. But what do I do now, do I just implement the function and send a PR?
